apps/prairielearn/package.json

```diff
@@ -193,6 +194,7 @@
     "@types/passport-azure-ad": "^4.3.1",
     "@types/pem": "^1.14.0",
     "@types/pg": "^8.10.2",
+    "@types/tabulator-tables": "~5.4.8",
```

Comment:
Apparently there is an older version 5.32.1 which was intended to come after 5.3.x and before 5.4.x, but is confusing semver/yarn. I'm using `~` instead of `^` to ensure yarn doesn't pick up that version.

Reply:
Did you report that to the DefinitelyTyped maintainers, or even the `tabulator-tables` folks? Sounds like they should unpublish the bad revisions.

Reply:
Done: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/66059

Reply:
Since the latest version is now 6.2.0, this is no longer a problem.